rendering problems at low zoom levels

jfirebaugh opened the pull request:
If you go to http://leaflet.apis.dev.openstreetmap.org/export and zoom out to the three lowest zoom levels, then click "Manually select a different area", you can see that the location filter has some strange effects:

![Screenshot](https://dl.dropbox.com/u/7078234/leaflet.png)

kajic commented:
That looks bad indeed. Wonder if it may have something to do with the fact that the filter seems to be spanning two latitude quadrants.

jfirebaugh committed:
Fix rendering at low zoom levels

At low zoom levels, the longitude coordinates of map bounds returned
by getBounds() may be outside of the normal ±180° range due to horizontal
wrapping. In order for the location filter to render correctly, it is
necessary to preserve these longitudes by passing true as the third
argument to all LatLng constructors.

jfirebaugh commented:
It turned out to be due to LatLng wrapping behavior.

kajic commented:
Thanks för the fix!
